tests/pages/contact.page.js

```diff
@@ -78,6 +79,86 @@ class ContactsPage {
     }
 
     // Additional method to verify the new contact in the list can be added here.
+
+    async verifyNewContactAdded(firstName) {
+        const verifyContact = '//*[@id="UsersCell-UserAddresses.Country-82b84172-8b20-40a4-b007-2c37068d7e36"]';
```

Comment:
The guid in this identifier is the id of a database item, this cannot be used as a static locator  because it always looks for a specific row not even relevant to the current testrun. so this guid has to be either passed to the verify function or cannot be used to identify the correct row. 
To verify if the contact has been added i would suggest to search for the entered last_name and first_name assuming they are unique for each test run